build.gradle

```
@@ -88,3 +109,10 @@ publishing {
 
 }
 
+jar {
+    manifest {
+        attributes(
+                "Version": System.getenv("GITHUB_RUN_NUMBER")
```

This might work better - should take the value from `version` variable
```suggestion
                "Version": "${project.version}"
```